clang/test/CheckedCRewriter/b10_allsafepointerstruct.c

```diff
@@ -23,12 +26,24 @@ struct r {
   struct r *next;
 };
 
+//CHECK_NOALL:   _Ptr<int> x;
+//CHECK_NOALL:   char *y;
+//CHECK_NOALL:   _Ptr<struct r> next;
+//CHECK_ALL:   _Ptr<int> x;
+//CHECK_ALL:   char *y;
+//CHECK_ALL:   _Ptr<struct r> next;
+
```

Comment:
I'm sorry to ask, but is it possible for these annotations to be added closer to the variables in question? I'd prefer to see
```
struct p {
  int *x;
//CHECK_NOALL:   _Ptr<int> x;
//CHECK_ALL:   _Ptr<int> x;
  char *y;
//CHECK_NOALL:   char *y;
//CHECK_ALL:   char *y;
};
struct r {
  int data;
  struct r *next;
//CHECK_NOALL:   _Ptr<struct r> next;
//CHECK_ALL:   _Ptr<struct r> next;
};

Reply:
For that matter: If you can make a single annotation `CHECK: ...` when the two are the same, so much the better. You had this with other tests previously, right?

Doing these things will make the tests easier to read and understand.

Reply:
This makes sense to me. It'll be a bit more complex but I think I can pull it off. 👍

Reply:
Looking over all of the output, it seems that the differences between using `-alltypes` and not using it are very slight. This makes me more inclined to say that it would be great to merge the annotations. I.e., for the above:

```
// RUN: cconv-standalone -alltypes %s -- | FileCheck -match-full-lines -check-prefixes="CHECK" %s
//RUN: cconv-standalone %s -- | FileCheck -match-full-lines -check-prefixes="CHECK" %s
// RUN: cconv-standalone %s -- | %clang -c -fcheckedc-extension -x c -o /dev/null -
...
struct p {
  int *x;
//CHECK:   _Ptr<int> x;
  char *y;
//CHECK:   char *y;
};
struct r {
  int data;
  struct r *next;
//CHECK:   _Ptr<struct r> next;
};
```
I think this is a lot easier to read.